src/tasks/AppleAppBuilder/Templates/main-simple.m

```diff
@@ -48,7 +48,6 @@ - (void)viewDidLoad {
     [button addTarget:self action:@selector(buttonClicked:) forControlEvents:UIControlEventTouchUpInside];
     [button setFrame:CGRectMake(50, 300, 200, 50)];
     [button setTitle:@"Click me (wire me up)" forState:UIControlStateNormal];
-    [button setExclusiveTouch:YES];
```

Comment:
Why are we removing this?

Reply:
`setExclusiveTouch` property is not supported on tvOS. It doesn't change tests functionality and we wanted to avoid `#ifdef` macros if possible.